sensor_array/helper/generate_breath_samples.py

```diff
@@ -127,3 +135,13 @@ def create_breath_samples(gases_dict, num_samples, filename=None, randomseed=101
             writer.writerows(comp_list_only_values)
 
     return comp_list
+
+
+def execute_create_breath_samples(config_file):
+    data = yaml_loader(config_file)
+    composition_set = data['Composition_Set']
+    num_samples = data['num_samples']
```

Comment:
definitely make consistent the capitalization of your YAML file or you will drive everybody crazy

Reply:
Will do! Need to go over this and make minor changes to follow pep8 after these PRs.